src/main/java/homework9/MobilePhone.java

```diff
@@ -0,0 +1,77 @@
+package homework9;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MobilePhone {
+    private String myNumber;
+    private ArrayList<Contact> myContacts;
+
+    public MobilePhone(String myNumber) {
+        this.myNumber = myNumber;
+        this.myContacts = new ArrayList<>();
+    }
+
+    public boolean addNewContact(Contact contact) {
+        if (findContact(contact.getName()) >= 0) {
+            System.out.println("Contact already exists.");
+            return false;
+        }
+
+        myContacts.add(contact);
+        return true;
+    }
+
+    public boolean updateContact(Contact oldContact, Contact newContact) {
+        int position = findContact(oldContact);
+        if (position < 0) {
+            System.out.println(oldContact.getName() + " was not found.");
+            return false;
+        }
+
+        myContacts.set(position, newContact);
+        System.out.println(oldContact.getName() + " was updated");
```

Comment:
`        System.out.println(oldContact.getName() + " was replaced with " + newContact.getName());`
would be more informative